src/components/BoardsList.tsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect } from "react";
+import fetchBoards from "../utils/fetchBoards";
+import BoardTile from "./BoardTile";
+import AddBoard from "./AddBoard";
+import BoardInfo from "../interfaces/Board";
+import BoardsListProps from "../interfaces/BoardListProps";
```

Comment:
A separate file for component props makes it hard to find and clutters the interface directory. We're also only ever importing that file in one place.